apps/laboratory/next.config.js

```diff
@@ -1,11 +1,34 @@
 /** @type {import('next').NextConfig} */
+const cspHeader = `
+  default-src 'self';
+  script-src 'self' 'unsafe-inline' 'unsafe-eval';
```

Comment:
Do we need `'unsafe-inline' 'unsafe-eval'`?

Reply:
Not for the deployed lab (or as a recommendation in general) I believe but omitting it will spam reports to the console when using the dev server:

<img width="738" alt="Screenshot 2024-06-11 at 17 24 00" src="https://github.com/WalletConnect/web3modal/assets/8663099/15611092-7af9-4c03-85c8-82a63ff785c0">

Reply:
We can conditionally add this to the CSP then